gateways/orbital/orbital.go

```diff
@@ -0,0 +1,122 @@
+package orbital
+
+import (
+	"bytes"
+	"encoding/xml"
+	"io/ioutil"
+	"net/http"
+	"strconv"
+
+	"github.com/BoltApp/sleet"
+	"github.com/BoltApp/sleet/common"
+)
+
+type Credentials struct {
+	username   string
+	password   string
+	merchantID int
+}
+
+type OrbitalClient struct {
+	host        string
+	credentials Credentials
+	httpClient  *http.Client
+}
+
+func NewClient(env common.Environment, credentials Credentials) *OrbitalClient {
```

Comment:
Are you planning on adding a `NewWithHttpClient` in a later PR? It seems like `NewClient` functions in most other gateways create a default http client and then call `NewWithHttpClient`

Reply:
+1 can we do the NewWithHttpClient?